pkg/oci/load_balancer_spec.go

```diff
@@ -30,19 +30,29 @@ import (
 	"github.com/oracle/oci-cloud-controller-manager/pkg/oci/util"
 )
 
+// SSLK8SSecret is a structure containing the data about a K8S secret required
+// to store SSL information required for BackendSets and Listeners
+type SSLK8SSecret struct {
```

Comment:
I'd call this `CertificateData`

Reply:
There's no harm in having our own local structure for representing ceritifcate data which shares a name IMO. I'd actually just call it `certificateData` though as we don't need to export it and it makes it clear that its an internal representation.